geoportailv3/__init__.py

```diff
@@ -28,6 +28,18 @@ def main(global_config, **settings):
         authentication_policy=create_authentication(settings)
     )
 
+    # overwrite print routes
+    config.add_route(
+        "lux_printproxy_report_create",
+        "/printproxy/report.{format}",
+        request_method="POST"
+    )
+    config.add_route(
+        "lux_printproxy_report_get",
+        "/printproxy/report/{ref}",
+        request_method="GET"
```

Comment:
For your information, I am currently adding print cancel support to ngeo and the client side of geoportailv3, so I need a print service that accepts cancel requests.